frameworks/FSharp/giraffe/src/App/Program.fs

```diff
@@ -0,0 +1,72 @@
+module App.App
+
+open System
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.Logging
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+open Microsoft.AspNetCore.Http
+
+// ---------------------------------
+// Models
+// ---------------------------------
+[<CLIMutable>]
+type JsonMessage = { message : string }
+
+// ---------------------------------
+// Web app
+// ---------------------------------
+
+let jsonUtf8 (data:obj) : HttpHandler =
+    fun (next : HttpFunc) (ctx : HttpContext) ->
+        let bytes = Utf8Json.JsonSerializer.Serialize(data)
+        ctx.SetContentType "application/json"
+        ctx.WriteBytesAsync bytes
+
+let webApp =
+    choose [
+        GET >=>
+            choose [
+                route "/plaintext" >=> text "Hello, World!"
+                route "/json" >=> json { message = "Hello, World!" }
+                route "/jsonutf8" >=> jsonUtf8 { message = "Hello, World!" }

```

Comment:
Will need to add a second dockerfile for this second json route (that's identical to first) `giraffe-utf8json.dockerfile`

Then add a new entry in `benchmark_config.json` for the json
```
"utf8json": {
    "json_url": "/jsonutf8",
   ...
```